app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name, age, wife=None, husband=None):
+        self.name = name
+        self.age = age
+
+        if husband is not None:
+            self.husband = husband
+        if wife is not None:
+            self.wife = wife
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    result_list = []
```

Comment:
You dont need result_list and you never used him in your code

Reply:
> You dont need result_list and you never used him in your code

And in your opinion, what do I have to do?